src/Azure.Functions.Cli/npm/lib/install.js

```diff
@@ -54,7 +54,7 @@ if (proxy) {
 
 https.get(options, response => {
 
-        const bar = new ProgressBar('[:bar] Downloading Azure Functions Cli', { 
+        const bar = new ProgressBar('[:bar] Downloading Azure Functions CLI', { 
```

Comment:
This can come later, but should we change this to "Downloading Azure Functions Core Tools"
@ahmelsayed @ahmedelnably

Reply:
I agree, that will make it more descriptive

Reply:
I can make that change if you'd like, just let me know.

Reply:
Be my guest :)

Reply:
Done